src/mantine-hooks/src/use-hover/use-hover.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { useHover } from './use-hover';
+
+const Target: React.FunctionComponent<any> = () => {
+  const { ref, hovered } = useHover();
+
+  return (
+    <div data-testid="target" ref={ref}>
+      {hovered ? 'true' : 'false'}
+    </div>
+  );
+};
+
+describe('@mantine/hooks/use-hover', () => {
+  render(<Target />);
+  const target = screen.getByTestId('target');
+  it('changes "hovered" on mouseenter/mouseleave correctly', () => {
+    expect(target).toHaveTextContent(/false/i);
```

Comment:
regex is not required, use strings instead.

Reply:
Thanks for the feedback! I have updated my request...